lib/node_modules/@stdlib/math/stats/anova1/lib/mean_table.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+// MODULES //
+
+var incrstdev = require( '@stdlib/math/stats/incr/stdev' );
+var uniqueElems = require('./unique_elems.js');
+
```

Comment:
Let's add `// MAIN //` header here.

Reply:
Added. I also added a space above `// MODULES //` to keep in convention with the two empty spaces.